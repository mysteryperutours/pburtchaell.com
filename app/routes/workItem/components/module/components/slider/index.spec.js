import React from 'react';
import Slider from './index';

describe(Slider.name, () => {
  it('renders component', () => {
    const component = (
      <Slider
        children={[{
          type: 'slide',
          title: 'Foo',
          subtitle: 'Foo 1',
          src: ''
        }, {
          type: 'slide',
          title: 'Bar',
          subtitle: 'Bar 1',
          src: ''
        }]}
      />
    );

    expect(shallow(component)).toMatchSnapshot();
  });
});
