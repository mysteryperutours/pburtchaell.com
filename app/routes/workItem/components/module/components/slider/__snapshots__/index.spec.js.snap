// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider renders component 1`] = `
<div
  className="image-slider-container"
  onClick={[Function]}
>
  <div
    className="image-slider"
    style={
      Object {
        "marginLeft": "-0px",
        "width": "1496px",
      }
    }
  >
    <div
      className="slide is-active"
      style={
        Object {
          "width": 688,
        }
      }
    >
      <div
        className="slide-header"
      >
        <div
          className="slide-header--title"
        >
          Foo
        </div>
        <div
          className="slide-header--subtitle"
        >
          Foo 1
        </div>
      </div>
      <div
        className="slide-image"
      >
        <div
          style={
            Object {
              "backgroundImage": "url()",
              "backgroundPosition": "center center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "contain",
              "height": "516px",
              "width": "100%",
            }
          }
        />
      </div>
    </div>
    <div
      className="slide is-left"
      style={
        Object {
          "width": 688,
        }
      }
    >
      <div
        className="slide-header"
      >
        <div
          className="slide-header--title"
        >
          Bar
        </div>
        <div
          className="slide-header--subtitle"
        >
          Bar 1
        </div>
      </div>
      <div
        className="slide-image"
      >
        <div
          style={
            Object {
              "backgroundImage": "url()",
              "backgroundPosition": "center center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "contain",
              "height": "516px",
              "width": "100%",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
